upgradeable-greeter/scripts/upgrade.js

```diff
@@ -0,0 +1,35 @@
+const { providerOverrides } = require('../utils/overrideProvider');
+const { ethers, upgrades } = require('hardhat');
+
+const proxyAddress = '0x721DbA5CE403DC1b04c7C2Bf8235761dDac1ebBb';
+
+async function main() {
+  const overrides = await providerOverrides();
+
+  const deployer = overrides.signer;
+
+  console.log('Upgrading contract with the account:', deployer.address);
+
+  console.log('Account balance:', (await deployer.getBalance()).toString());
+
+  const GreeterV2 = await ethers.getContractFactory('GreeterV2', deployer);
+  const instance = await upgrades.upgradeProxy(proxyAddress, GreeterV2);
+
+  console.log('GreeterV2 address:', instance.address);
+
+  const originalGreeting = await instance.greet();
+
+  await instance.setGreetingV2('Konichiwa, Kakarot!');
```

Comment:
I like this lol

Reply:
Gotta keep it in line with the theme 😅 

![image](https://user-images.githubusercontent.com/8097439/170006994-a39d01cc-8d01-4e99-b146-de0e11149e03.png)